File: src/CloudMining.Domain/Models/ShareablePayment.cs
using CloudMining.Domain.Enums;
using CloudMining.Domain.Models.Base;

namespace CloudMining.Domain.Models
{
	public class ShareablePayment : Payment
	{
		public DateTime Date { get; set; }
		public PaymentType Type { get; set; }
		public bool IsCompleted { get; set; }
		public List<PaymentShare> PaymentShares { get; set; }
	}
}

File: src/appVue/src/models/ShareablePayment.ts
import { CurrencyCode } from '@/enums/CurrencyCode';
import { PaymentType } from '@/enums/PaymentType';

export interface ShareablePayment{
    caption: string | null;
    currencyCode: CurrencyCode;
    paymentType: PaymentType;
    date: Date;
    amount: number;
}


